source/elm-live.js

```diff
@@ -112,7 +112,7 @@ ${ indent(String(elmMake.error), '  ') }
   }
 
   // Watch Elm files
-  const watcher = chokidar.watch('**/*.elm', { ignoreInitial: false, ignorePermissionErrors: true });
+  const watcher = chokidar.watch('**/*.elm', { ignoreInitial: true, ignorePermissionErrors: true });
```

Comment:
Why do we need `ignorePermissionErrors`? If it’s important, could you please add a regression test to make sure no-one breaks the behavior in the future?

As well as that, please add a test for `ignoreInitial` for the same reasons.